assets/scripts/preinit/system_capabilities.js

```diff
@@ -25,7 +25,11 @@ export const system = {
   pageVisibility: false,
   hiddenProperty: false,
   visibilityState: false,
-  visibilityChange: false
+  visibilityChange: false,
+  unsupportedBrowser: {
+    name: 'ie',
+    version: '8'
+  }
```

Comment:
This is actually not a great place to put it, but I do understand why it seemed like it. We'll need to move this elsewhere.